pkg/splunk/enterprise/finalizers.go

```diff
@@ -31,27 +31,32 @@ func init() {
 
 // DeleteSplunkPvc removes all corresponding PersistentVolumeClaims that are associated with a custom resource.
 func DeleteSplunkPvc(cr splcommon.MetaObject, c splcommon.ControllerClient) error {
-	scopedLog := log.WithName("DeleteSplunkPvc").WithValues("kind", cr.GetObjectKind().GroupVersionKind().Kind,
+	var objectKind string
+	objectKind = cr.GetObjectKind().GroupVersionKind().Kind
+
+	scopedLog := log.WithName("DeleteSplunkPvc").WithValues("kind", objectKind,
 		"name", cr.GetName(), "namespace", cr.GetNamespace())
 
 	var component string
-	switch cr.GetObjectKind().GroupVersionKind().Kind {
+	switch objectKind {
 	case "Standalone":
 		component = "standalone"
 	case "LicenseMaster":
 		component = "license-master"
 	case "SearchHeadCluster":
 		component = "search-head"
-	case "IndexerCluster", "ClusterMaster":
+	case "IndexerCluster":
 		component = "indexer"
+	case "ClusterMaster":
+		component = "cluster-master"
 	default:
 		scopedLog.Info("Skipping PVC removal")
 		return nil
 	}
 
 	// get list of PVCs for this cluster
 	labels := map[string]string{
-		"app.kubernetes.io/part-of": fmt.Sprintf("splunk-%s-%s", cr.GetName(), component),
+		"app.kubernetes.io/instance": fmt.Sprintf("splunk-%s-%s", cr.GetName(), component),
```

Comment:
Just a suggestion for now or in the future: 

To maintain consistency, it might be good to use the same API to set/retrieve labels(GetSplunkLabels). We could modify GetSplunkLabels's prototype to **getSplunkLabels(instanceIdentifier string, instanceType InstanceType, partOfIdentifier string, selectFew []string)** and call it using **var sfew []string = "instance"** as the selectFew parameter.

Reply:
Probably will take it for future improvement.